src/transformers/utils/fx.py

```diff
@@ -115,8 +115,8 @@ def _generate_supported_model_classes(
     # TODO: add support for them as it should be quite easy to do so (small blocking issues).
     # XLNetForQuestionAnswering,
 ]
-_SUPPORTED_MODELS = tuple(_REGULAR_SUPPORTED_MODELS + _SPECIAL_SUPPORTED_MODELS)
-
+_SUPPORTED_MODELS = tuple(
+    sorted(list(set(_REGULAR_SUPPORTED_MODELS + _SPECIAL_SUPPORTED_MODELS)), key=lambda c: c.__name__))
```

Comment:
Why do you need it sorted?

Reply:
just to make it more readable in the error message

Reply:
Alright, got it, thanks!